plugins/Jobs.cpp

```diff
@@ -367,8 +372,9 @@ bool BedrockPlugin_Jobs::processCommand(BedrockNode* node, SQLite& db, BedrockNo
         // Update the data
         if (!db.write("UPDATE jobs "
                       "SET data=" +
-                      SQ(request["data"]) + " "
-                                            "WHERE jobID=" +
+                      SQ(request["data"]) + " " +
+                      (request.isSet("repeat") ? ", repeat=" + SQ(SToUpper(request["repeat"])) : "") +
```

Comment:
I think this needs to be ` (request.isSet("repeat") ? ", repeat=" + SQ(SToUpper(request["repeat"])) : "''")` If you don't pass repeat, wouldn't you end up with: ` repeat = WHERE jobID = 1`, right?